web/client/epics/featuregrid.js

```diff
@@ -235,11 +235,9 @@ const addPagination = (filterObj, pagination) => ({
 const createChangesTransaction = (changes, newFeatures, {insert, update, propertyChange, getPropertyName, transaction})=>
```

Comment:
can you create a dedicated test for checking the body of the transaction payload?

Reply:
In the epic test file ?